lib/spack/docs/basic_usage.rst

```diff
@@ -1268,7 +1268,7 @@ This script may be directly executed in bash via:
    source < (spack module find tcl --dependencies --shell py-numpy git)
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^
-Regenerating Module files
+Generating Module files
 ^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
There were two headings labelled `Regenerating Module Files`. I think the first one should be `Generating Module Files` in the context that it was in.